File: TraversalLL/traversalLL.c
#include<stdio.h>
#include<stdlib.h>

typedef struct Node {
    int data;
    struct Node * next;
}Node;

void traverseAndPrint(Node *head){
    Node * currentNode = head;
    while(currentNode){
        printf("%d -> ", currentNode->data);
        currentNode = currentNode->next;
    }
    printf("null\n");
}
int main(){
    Node *node1 = (Node*)malloc(sizeof(Node));
    Node *node2 = (Node*)malloc(sizeof(Node));
    Node *node3 = (Node*)malloc(sizeof(Node));
    Node *node4 = (Node*)malloc(sizeof(Node));
    Node *node5 = (Node*)malloc(sizeof(Node));
   
    node1->data = 7;
    node2->data = 9;
    node3->data = 12;
    node4->data = 34;
    node5->data = 67;

    node1->next = node2;
    node2->next = node3;
    node3->next = node4;
    node4->next = node5;
    node5->next = NULL;

    traverseAndPrint(node1);


    free(node1);
    free(node2);
    free(node3);
    free(node4);
    free(node5);

    return 0;
}

File: TraversalLL/traversalLL.py
class Node:
    def __init__(self,data):
        self.data = data
        self.next = None

def traverseAndPrint(head):
    currentNode = head
    while currentNode:
        print(currentNode.data,  end=' -> ')
        currentNode = currentNode.next
    print('null')

node1 = Node(8)
node2 = Node(9)
node3 = Node(11)
node4 = Node(16)
node5 = Node(23)

node1.next = node2
node2.next = node3
node3.next = node4
node4.next = node5

traverseAndPrint(node1)